In [2]:
import pandas as pd
import numpy as np

Importing the data as a dataframe.

In [3]:
dataset = pd.read_csv("data/Data_Lv3_UMDOTS_Escooters.csv")

Lets take a look at a few data point

In [4]:
#dataset.head()
dataset.tail()

Unnamed: 0  Unnamed: 0.1                    CREATED  \
40320       40320         40320  2020-10-24 22:21:40+00:00   
40321       40321         40321  2020-10-24 21:28:43+00:00   
40322       40322         40322  2020-10-25 01:22:00+00:00   
40323       40323         40323  2020-10-25 00:55:24+00:00   
40324       40324         40324  2020-10-24 06:03:05+00:00   

                           START                        END  DISTANCE  \
40320  2020-10-24 22:21:40+00:00  2020-10-24 22:26:38+00:00     0.794   
40321  2020-10-24 21:28:43+00:00  2020-10-24 21:32:54+00:00     0.404   
40322  2020-10-25 01:22:00+00:00  2020-10-25 01:26:16+00:00     0.629   
40323  2020-10-25 00:55:24+00:00  2020-10-25 01:07:38+00:00     0.567   
40324  2020-10-24 06:03:05+00:00  2020-10-24 06:38:05+00:00     5.220   

       RIDE_ID  CUSTOMER_ID  MINUTES VEHICLE_TYPE  \
40320  3304114       322401        5      Scooter   
40321  3303630       298095        5      Scooter   
40322  3305589       604319        5      Scooter   
40323  3305336       610943       13      Scooter   
40324  3299791       324757       35      Scooter   

                                                    PATH  \
40320  {\n  "coordinates": [\n    [\n      -76.943936...   
40321  {\n  "coordinates": [\n    [\n      -76.935917...   
40322  {\n  "coordinates": [\n    [\n      -76.940231...   
40323  {\n  "coordinates": [\n    [\n      -76.944152...   
40324  {\n  "coordinates": [\n    [\n      -76.934166...   

                                              TIMESTAMPS  START LONG  \
40320  [\n  "2020-10-24T22:21:46Z",\n  "2020-10-24T22...  -76.943937   
40321  [\n  "2020-10-24T21:28:49Z",\n  "2020-10-24T21...  -76.935917   
40322  [\n  "2020-10-25T01:22:07Z",\n  "2020-10-25T01...  -76.940231   
40323  [\n  "2020-10-25T00:55:30Z",\n  "2020-10-25T00...  -76.944153   
40324  [\n  "2020-10-24T06:03:12Z",\n  "2020-10-24T06...  -76.934167   

       START LAT   END LONG    END LAT  
40320  38.988110 -76.934104  38.992061  
40321  38.986389 -76.933818  38.992196  
40322  38.996377 -76.945600  38.992842  
40323  38.991956 -76.940216  38.987199  
40324  38.990645 -76.936088  38.982452

In [5]:
dataset.shape

(40325, 16)

In [6]:
dataset.dtypes

Unnamed: 0        int64
Unnamed: 0.1      int64
CREATED          object
START            object
END              object
DISTANCE        float64
RIDE_ID           int64
CUSTOMER_ID       int64
MINUTES           int64
VEHICLE_TYPE     object
PATH             object
TIMESTAMPS       object
START LONG      float64
START LAT       float64
END LONG        float64
END LAT         float64
dtype: object

In [7]:
# All the data types do not match lets fix them

# Convert as Timestamp data
dataset["CREATED"] = pd.to_datetime(dataset["CREATED"])
# Convert as Timestamp data
dataset["START"] = pd.to_datetime(dataset["START"])
# Convert as Timestamp data
dataset["END"] = pd.to_datetime(dataset["END"])
# Convert timestamp as list
dataset["TIMESTAMPS"] = dataset["TIMESTAMPS"].apply(eval)

The dataset has 40325 rows and 16 columns

In [8]:
# Get the year of each data entry
# Initialize new columns
dataset["Year"] = dataset["CREATED"][0].year
for index, item in dataset.iterrows():
    dataset["Year"][index] = dataset["CREATED"][index].year

<ipython-input-8-71b62617ba7d>:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  dataset["Year"][index] = dataset["CREATED"][index].year


In [9]:
ride_count_year = dataset.groupby("Year").size().reset_index(name='ride_count')
ride_count_year

Year  ride_count
0  2019       18382
1  2020       21943

The distance column is missing values

In [10]:
dataset["DISTANCE"].isna().sum()
#dataset["DISTANCE"].dropna()

2273

Total of 2273 missing datapoint for distance. No sure how to deal with this yet.

Next we separate all ride by years: October 2020 and October 2019

In [11]:
dataset.groupby("CREATED")

In [12]:

dataset["START POINT"] = (dataset["START LAT"].astype(str)) + ", " + dataset["START LONG"].astype(str)

dataset["END POINT"] = (dataset["END LAT"].astype(str)) + ", " + dataset["END LONG"].astype(str)
dataset.head()

Unnamed: 0  Unnamed: 0.1                   CREATED  \
0           0             0 2020-10-03 03:32:15+00:00   
1           1             1 2020-10-03 03:07:39+00:00   
2           2             2 2020-10-03 03:17:05+00:00   
3           3             3 2020-10-03 02:24:58+00:00   
4           4             4 2020-10-03 03:19:34+00:00   

                      START                       END  DISTANCE  RIDE_ID  \
0 2020-10-03 03:32:15+00:00 2020-10-03 03:32:46+00:00     0.008  3135665   
1 2020-10-03 03:07:39+00:00 2020-10-03 03:37:06+00:00     1.062  3135395   
2 2020-10-03 03:17:05+00:00 2020-10-03 03:24:24+00:00     0.847  3135503   
3 2020-10-03 02:24:58+00:00 2020-10-03 03:11:56+00:00     3.447  3134955   
4 2020-10-03 03:19:34+00:00 2020-10-03 03:22:45+00:00     0.386  3135529   

   CUSTOMER_ID  MINUTES VEHICLE_TYPE  \
0       193964        1      Scooter   
1       542947       30      Scooter   
2       517321        8      Scooter   
3       578051       47      Scooter   
4       552305        4      Scooter   

                                                PATH  \
0  {\n  "coordinates": [\n    [\n      -76.934163...   
1  {\n  "coordinates": [\n    [\n      -76.938180...   
2  {\n  "coordinates": [\n    [\n      -76.937599...   
3  {\n  "coordinates": [\n    [\n      -76.943553...   
4  {\n  "coordinates": [\n    [\n      -76.938115...   

                                          TIMESTAMPS  START LONG  START LAT  \
0       [2020-10-03T03:32:14Z, 2020-10-03T03:32:30Z]  -76.934163  38.990656   
1  [2020-10-03T03:07:46Z, 2020-10-03T03:12:16Z, 2...  -76.938181  38.982087   
2  [2020-10-03T03:17:12Z, 2020-10-03T03:17:32Z, 2...  -76.937600  38.979613   
3  [2020-10-03T02:25:04Z, 2020-10-03T02:26:24Z, 2...  -76.943553  38.982470   
4  [2020-10-03T03:19:40Z, 2020-10-03T03:19:45Z, 2...  -76.938116  38.982084   

    END LONG    END LAT  Year                             START POINT  \
0 -76.934285  38.990739  2020                  38.9906555, -76.934163   
1 -76.949165  38.992849  2020  38.982087166666666, -76.93818083333333   
2 -76.933335  38.992676  2020   38.97961266666667, -76.93759983333334   
3 -76.943546  38.982476  2020  38.982470166666666, -76.94355316666666   
4 -76.943257  38.986101  2020          38.9820835, -76.93811566666666   

                                END POINT  
0  38.990738666666665, -76.93428483333334  
1   38.99284866666667, -76.94916483333333  
2           38.992676, -76.93333466666667  
3  38.982475666666666, -76.94354633333333  
4                   38.986101, -76.943257

2020    21943
2019    18382
Name: Year, dtype: int64

In [45]:
with open('json_column', 'w') as f:
    json.dump(dataset_filtered[dataset_filtered['PATH']], f)


KeyError: 'None of [Index([\'{\\n  "coordinates": [\\n    [\\n      -76.934163,\\n      38.9906555\\n    ],\\n    [\\n      -76.93428483333334,\\n      38.990738666666665\\n    ]\\n  ],\\n  "type": "MultiPoint"\\n}\',\n       \'{\\n  "coordinates": [\\n    [\\n      -76.93818083333333,\\n      38.982087166666666\\n    ],\\n    [\\n      -76.93838233333334,\\n      38.98218333333333\\n    ],\\n    [\\n      -76.93821283333334,\\n      38.98213833333333\\n    ],\\n    [\\n      -76.9383925,\\n      38.98218966666666\\n    ],\\n    [\\n      -76.93861233333334,\\n      38.982253166666666\\n    ],\\n    [\\n      -76.9387355,\\n      38.98229416666667\\n    ],\\n    [\\n      -76.93886233333333,\\n      38.9823245\\n    ],\\n    [\\n      -76.93901266666667,\\n      38.982361\\n    ],\\n    [\\n      -76.93918733333334,\\n      38.98240466666667\\n    ],\\n    [\\n      -76.9393465,\\n      38.982432333333335\\n    ],\\n    [\\n      -76.93950883333333,\\n      38.98248916666667\\n    ],\\n    [\\n      -76.9395875,\\n      38.98258166666667\\n    ],\\n    [\\n      -76.93972416666666,\\n      38.98275983333333\\n    ],\\n    [\\n      -76.93982066666666,\\n      38.982890166666664\\n    ],\\n    [\\n      -76.93988033333333,\\n      38.98298966666667\\n    ],\\n    [\\n      -76.93993766666667,\\n      38.983129166666664\\n    ],\\n    [\\n      -76.93997516666667,\\n      38.98324183333333\\n    ],\\n    [\\n      -76.94000333333334,\\n      38.98334333333333\\n    ],\\n    [\\n      -76.9400125,\\n      38.98352116666667\\n    ],\\n    [\\n      -76.93997766666666,\\n      38.983636\\n    ],\\n    [\\n      -76.939978,\\n      38.98376183333333\\n    ],\\n    [\\n      -76.93998616666667,\\n      38.98389133333333\\n    ],\\n    [\\n      -76.93999966666667,\\n      38.984015666666664\\n    ],\\n    [\\n      -76.940021,\\n      38.984249\\n    ],\\n    [\\n      -76.94001433333334,\\n      38.98442166666667\\n    ],\\n    [\\n      -76.9400205,\\n      38.98455766666667\\n    ],\\n    [\\n      -76.94002533333334,\\n      38.98468316666667\\n    ],\\n    [\\n      -76.94000316666667,\\n      38.9848495\\n    ],\\n    [\\n      -76.93999583333333,\\n      38.9850015\\n    ],\\n    [\\n      -76.93998766666667,\\n      38.985114833333334\\n    ],\\n    [\\n      -76.93998533333334,\\n      38.98521616666667\\n    ],\\n    [\\n      -76.94000683333333,\\n      38.98535533333333\\n    ],\\n    [\\n      -76.9400085,\\n      38.985542833333334\\n    ],\\n    [\\n      -76.940008,\\n      38.98572516666667\\n    ],\\n    [\\n      -76.94000416666667,\\n      38.985837333333336\\n    ],\\n    [\\n      -76.94002266666666,\\n      38.98594683333334\\n    ],\\n    [\\n      -76.939999,\\n      38.986131\\n    ],\\n    [\\n      -76.94000483333333,\\n      38.986255666666665\\n    ],\\n    [\\n      -76.93998666666667,\\n      38.98638783333333\\n    ],\\n    [\\n      -76.94000866666667,\\n      38.9865575\\n    ],\\n    [\\n      -76.94001533333333,\\n      38.9867565\\n    ],\\n    [\\n      -76.94001316666666,\\n      38.9868585\\n    ],\\n    [\\n      -76.94006333333333,\\n      38.98698816666667\\n    ],\\n    [\\n      -76.94007616666667,\\n      38.98711466666666\\n    ],\\n    [\\n      -76.940046,\\n      38.98723433333333\\n    ],\\n    [\\n      -76.94002166666667,\\n      38.987391333333335\\n    ],\\n    [\\n      -76.94015716666667,\\n      38.9874625\\n    ],\\n    [\\n      -76.94035283333334,\\n      38.987506\\n    ],\\n    [\\n      -76.94052683333334,\\n      38.987593333333336\\n    ],\\n    [\\n      -76.94066583333333,\\n      38.987675333333335\\n    ],\\n    [\\n      -76.94068,\\n      38.9878565\\n    ],\\n    [\\n      -76.94068783333333,\\n      38.987965333333335\\n    ],\\n    [\\n      -76.94063133333333,\\n      38.988133166666664\\n    ],\\n    [\\n      -76.94065666666667,\\n      38.98825983333333\\n    ],\\n    [\\n      -76.94068766666666,\\n      38.988393\\n    ],\\n    [\\n      -76.940695,\\n      38.988543666666665\\n    ],\\n    [\\n      -76.94071,\\n      38.9886725\\n    ],\\n    [\\n      -76.94072683333333,\\n      38.9887885\\n    ],\\n    [\\n      -76.940772,\\n      38.9889655\\n    ],\\n    [\\n      -76.94079566666667,\\n      38.9891105\\n    ],\\n    [\\n      -76.940771,\\n      38.98922316666667\\n    ],\\n    [\\n      -76.94075266666667,\\n      38.98932416666667\\n    ],\\n    [\\n      -76.94072983333334,\\n      38.98948633333333\\n    ],\\n    [\\n      -76.94074316666666,\\n      38.9896095\\n    ],\\n    [\\n      -76.94075483333333,\\n      38.989749833333335\\n    ],\\n    [\\n      -76.94079083333334,\\n      38.9898945\\n    ],\\n    [\\n      -76.940796,\\n      38.99002383333333\\n    ],\\n    [\\n      -76.94088316666667,\\n      38.99011016666667\\n    ],\\n    [\\n      -76.941097,\\n      38.990161\\n    ],\\n    [\\n      -76.94124616666667,\\n      38.990225\\n    ],\\n    [\\n      -76.9413915,\\n      38.990293333333334\\n    ],\\n    [\\n      -76.94151833333333,\\n      38.990355\\n    ],\\n    [\\n      -76.94163066666667,\\n      38.99045183333333\\n    ],\\n    [\\n      -76.94174216666667,\\n      38.99051\\n    ],\\n    [\\n      -76.9419295,\\n      38.9905545\\n    ],\\n    [\\n      -76.94208233333333,\\n      38.990566\\n    ],\\n    [\\n      -76.94213116666667,\\n      38.990661\\n    ],\\n    [\\n      -76.94228583333333,\\n      38.9906615\\n    ],\\n    [\\n      -76.9425455,\\n      38.9906485\\n    ],\\n    [\\n      -76.942744,\\n      38.9906485\\n    ],\\n    [\\n      -76.94291833333334,\\n      38.990656666666666\\n    ],\\n    [\\n      -76.94309166666666,\\n      38.9906575\\n    ],\\n    [\\n      -76.94323016666667,\\n      38.9906505\\n    ],\\n    [\\n      -76.943423,\\n      38.990658\\n    ],\\n    [\\n      -76.94361516666666,\\n      38.99063133333333\\n    ],\\n    [\\n      -76.94381916666667,\\n      38.990630833333334\\n    ],\\n    [\\n      -76.94398583333333,\\n      38.990641833333335\\n    ],\\n    [\\n      -76.94416116666666,\\n      38.99063733333333\\n    ],\\n    [\\n      -76.9443915,\\n      38.990634666666665\\n    ],\\n    [\\n      -76.94465783333334,\\n      38.99063366666667\\n    ],\\n    [\\n      -76.94495283333333,\\n      38.9906405\\n    ],\\n    [\\n      -76.9451885,\\n      38.99066116666667\\n    ],\\n    [\\n      -76.945315,\\n      38.99068466666667\\n    ],\\n    [\\n      -76.945561,\\n      38.99068366666667\\n    ],\\n    [\\n      -76.945427,\\n      38.99072316666667\\n    ],\\n    [\\n      -76.94564083333333,\\n      38.9907\\n    ],\\n    [\\n      -76.945494,\\n      38.990679166666666\\n    ],\\n    [\\n      -76.94561566666667,\\n      38.990762333333336\\n    ],\\n    [\\n      -76.94578916666667,\\n      38.9908235\\n    ],\\n    [\\n      -76.9459505,\\n      38.99089716666667\\n    ],\\n    [\\n      -76.94613283333334,\\n      38.99097166666667\\n    ],\\n    [\\n      -76.94630866666667,\\n      38.99105483333334\\n    ],\\n    [\\n      -76.94644,\\n      38.99112383333333\\n    ],\\n    [\\n      -76.946609,\\n      38.99121366666667\\n    ],\\n    [\\n      -76.94681366666667,\\n      38.99132866666667\\n    ],\\n    [\\n      -76.9469705,\\n      38.99139816666667\\n    ],\\n    [\\n      -76.94708183333333,\\n      38.99148983333333\\n    ],\\n    [\\n      -76.94717916666667,\\n      38.991587833333334\\n    ],\\n    [\\n      -76.947302,\\n      38.991686666666666\\n    ],\\n    [\\n      -76.94739366666667,\\n      38.9917885\\n    ],\\n    [\\n      -76.94751683333334,\\n      38.991892166666666\\n    ],\\n    [\\n      -76.94761316666667,\\n      38.991991166666665\\n    ],\\n    [\\n      -76.9476865,\\n      38.99208633333333\\n    ],\\n    [\\n      -76.94789566666667,\\n      38.992248333333336\\n    ],\\n    [\\n      -76.94805033333333,\\n      38.992301\\n    ],\\n    [\\n      -76.94828016666666,\\n      38.99234916666666\\n    ],\\n    [\\n      -76.94842633333333,\\n      38.992377166666664\\n    ],\\n    [\\n      -76.94853316666666,\\n      38.99247583333333\\n    ],\\n    [\\n      -76.948498,\\n      38.99259133333334\\n    ],\\n    [\\n      -76.94847933333334,\\n      38.9927095\\n    ],\\n    [\\n      -76.94853116666667,\\n      38.992833166666664\\n    ],\\n    [\\n      -76.94867183333334,\\n      38.99288033333333\\n    ],\\n    [\\n      -76.94878733333333,\\n      38.992936666666665\\n    ],\\n    [\\n      -76.94892166666666,\\n      38.99298616666667\\n    ],\\n    [\\n      -76.94910383333334,\\n      38.992945666666664\\n    ],\\n    [\\n      -76.94916483333333,\\n      38.99284866666667\\n    ]\\n  ],\\n  "type": "MultiPoint"\\n}\',\n       \'{\\n  "coordinates": [\\n    [\\n      -76.93759983333334,\\n      38.97961266666667\\n    ],\\n    [\\n      -76.93777183333333,\\n      38.97969\\n    ],\\n    [\\n      -76.93796,\\n      38.979699\\n    ],\\n    [\\n      -76.93804183333333,\\n      38.979789833333335\\n    ],\\n    [\\n      -76.93804233333333,\\n      38.97991716666667\\n    ],\\n    [\\n      -76.93804716666666,\\n      38.980117666666665\\n    ],\\n    [\\n      -76.93806683333334,\\n      38.98032666666667\\n    ],\\n    [\\n      -76.93806933333333,\\n      38.980543\\n    ],\\n    [\\n      -76.93806316666667,\\n      38.9807\\n    ],\\n    [\\n      -76.938056,\\n      38.980858\\n    ],\\n    [\\n      -76.93804816666666,\\n      38.98096433333333\\n    ],\\n    [\\n      -76.93806466666666,\\n      38.981073333333335\\n    ],\\n    [\\n      -76.93805316666666,\\n      38.98120133333333\\n    ],\\n    [\\n      -76.93803966666667,\\n      38.981337\\n    ],\\n    [\\n      -76.93802133333334,\\n      38.981481\\n    ],\\n    [\\n      -76.9380325,\\n      38.98167566666667\\n    ],\\n    [\\n      -76.93806016666667,\\n      38.981938166666666\\n    ],\\n    [\\n      -76.938107,\\n      38.98207583333333\\n    ],\\n    [\\n      -76.9381055,\\n      38.982218833333334\\n    ],\\n    [\\n      -76.93806216666667,\\n      38.982376333333335\\n    ],\\n    [\\n      -76.93804833333333,\\n      38.982536\\n    ],\\n    [\\n      -76.9380455,\\n      38.9827365\\n    ],\\n    [\\n      -76.93803066666666,\\n      38.982941333333336\\n    ],\\n    [\\n      -76.9379995,\\n      38.983156666666666\\n    ],\\n    [\\n      -76.93796916666666,\\n      38.983371\\n    ],\\n    [\\n      -76.93792533333334,\\n      38.98359283333333\\n    ],\\n    [\\n      -76.93787366666666,\\n      38.98382683333333\\n    ],\\n    [\\n      -76.93781066666666,\\n      38.98407\\n    ],\\n    [\\n      -76.937741,\\n      38.98432016666667\\n    ],\\n    [\\n      -76.93765883333333,\\n      38.984563333333334\\n    ],\\n    [\\n      -76.93758633333333,\\n      38.9847685\\n    ],\\n    [\\n      -76.93750483333334,\\n      38.984979\\n    ],\\n    [\\n      -76.93738333333333,\\n      38.985200666666664\\n    ],\\n    [\\n      -76.9372775,\\n      38.98543016666667\\n    ],\\n    [\\n      -76.9371935,\\n      38.985605166666666\\n    ],\\n    [\\n      -76.9370925,\\n      38.98579316666667\\n    ],\\n    [\\n      -76.936983,\\n      38.985978833333334\\n    ],\\n    [\\n      -76.9368975,\\n      38.986122\\n    ],\\n    [\\n      -76.9368035,\\n      38.98628733333334\\n    ],\\n    [\\n      -76.93667783333333,\\n      38.98649016666667\\n    ],\\n    [\\n      -76.93655966666667,\\n      38.98671616666667\\n    ],\\n    [\\n      -76.93640683333334,\\n      38.98696066666667\\n    ],\\n    [\\n      -76.93624516666667,\\n      38.987212\\n    ],\\n    [\\n      -76.93612216666666,\\n      38.987402\\n    ],\\n    [\\n      -76.93604583333334,\\n      38.98753333333333\\n    ],\\n    [\\n      -76.935987,\\n      38.98763066666667\\n    ],\\n    [\\n      -76.93583583333333,\\n      38.98784066666666\\n    ],\\n    [\\n      -76.93580816666666,\\n      38.9879815\\n    ],\\n    [\\n      -76.93568683333334,\\n      38.98810116666667\\n    ],\\n    [\\n      -76.93557183333333,\\n      38.98826866666667\\n    ],\\n    [\\n      -76.93546366666666,\\n      38.988473166666665\\n    ],\\n    [\\n      -76.93534333333334,\\n      38.98871466666667\\n    ],\\n    [\\n      -76.93520666666667,\\n      38.98885383333333\\n    ],\\n    [\\n      -76.93509533333334,\\n      38.989006333333336\\n    ],\\n    [\\n      -76.934986,\\n      38.9891705\\n    ],\\n    [\\n      -76.93487233333333,\\n      38.98935016666667\\n    ],\\n    [\\n      -76.93476966666667,\\n      38.989521\\n    ],\\n    [\\n      -76.9346665,\\n      38.98969366666667\\n    ],\\n    [\\n      -76.93458566666666,\\n      38.989826666666666\\n    ],\\n    [\\n      -76.934532,\\n      38.989920166666664\\n    ],\\n    [\\n      -76.93450666666666,\\n      38.99007016666667\\n    ],\\n    [\\n      -76.93438366666666,\\n      38.99029066666667\\n    ],\\n    [\\n      -76.93425816666667,\\n      38.990498333333335\\n    ],\\n    [\\n      -76.93416366666666,\\n      38.990685166666665\\n    ],\\n    [\\n      -76.9341005,\\n      38.99081183333333\\n    ],\\n    [\\n      -76.93405,\\n      38.99095766666667\\n    ],\\n    [\\n      -76.933978,\\n      38.991119833333336\\n    ],\\n    [\\n      -76.93402566666667,\\n      38.991294833333335\\n    ],\\n    [\\n      -76.9339365,\\n      38.99141516666667\\n    ],\\n    [\\n      -76.93391633333333,\\n      38.99157666666667\\n    ],\\n    [\\n      -76.93392766666666,\\n      38.99170933333333\\n    ],\\n    [\\n      -76.93382016666666,\\n      38.99183083333333\\n    ],\\n    [\\n      -76.933654,\\n      38.991992333333336\\n    ],\\n    [\\n      -76.93355116666666,\\n      38.99216033333333\\n    ],\\n    [\\n      -76.93355216666667,\\n      38.99228266666667\\n    ],\\n    [\\n      -76.93351133333333,\\n      38.99241883333333\\n    ],\\n    [\\n      -76.93339166666667,\\n      38.99254833333333\\n    ],\\n    [\\n      -76.93333466666667,\\n      38.992676\\n    ]\\n  ],\\n  "type": "MultiPoint"\\n}\',\n       \'{\\n  "coordinates": [\\n    [\\n      -76.94355316666666,\\n      38.982470166666666\\n    ],\\n    [\\n      -76.9438835,\\n      38.982395833333335\\n    ],\\n    [\\n      -76.94351366666666,\\n      38.98245933333333\\n    ],\\n    [\\n      -76.9437775,\\n      38.982394166666666\\n    ],\\n    [\\n      -76.94362833333334,\\n      38.982614833333336\\n    ],\\n    [\\n      -76.94353,\\n      38.9824185\\n    ],\\n    [\\n      -76.94371033333333,\\n      38.98249933333334\\n    ],\\n    [\\n      -76.94353583333333,\\n      38.9824465\\n    ],\\n    [\\n      -76.9434805,\\n      38.9820865\\n    ],\\n    [\\n      -76.94367583333333,\\n      38.98176733333333\\n    ],\\n    [\\n      -76.94406866666667,\\n      38.98183483333333\\n    ],\\n    [\\n      -76.9444295,\\n      38.98188416666667\\n    ],\\n    [\\n      -76.94481916666666,\\n      38.982259\\n    ],\\n    [\\n      -76.9448265,\\n      38.982585666666665\\n    ],\\n    [\\n      -76.94482466666666,\\n      38.98306133333333\\n    ],\\n    [\\n      -76.94506983333333,\\n      38.98345283333333\\n    ],\\n    [\\n      -76.94509883333333,\\n      38.983835166666665\\n    ],\\n    [\\n      -76.94516716666666,\\n      38.984212666666664\\n    ],\\n    [\\n      -76.94551633333333,\\n      38.984558\\n    ],\\n    [\\n      -76.945909,\\n      38.9848225\\n    ],\\n    [\\n      -76.9461775,\\n      38.98499133333333\\n    ],\\n    [\\n      -76.9465065,\\n      38.98516016666667\\n    ],\\n    [\\n      -76.9468075,\\n      38.985436666666665\\n    ],\\n    [\\n      -76.94694883333334,\\n      38.98573016666667\\n    ],\\n    [\\n      -76.94694933333334,\\n      38.98599316666667\\n    ],\\n    [\\n      -76.946896,\\n      38.98628983333333\\n    ],\\n    [\\n      -76.94676633333333,\\n      38.98657733333334\\n    ],\\n    [\\n      -76.94658166666666,\\n      38.986872\\n    ],\\n    [\\n      -76.94639383333333,\\n      38.987164666666665\\n    ],\\n    [\\n      -76.94581083333334,\\n      38.9872415\\n    ],\\n    [\\n      -76.9454355,\\n      38.98750066666667\\n    ],\\n    [\\n      -76.9454105,\\n      38.987838833333335\\n    ],\\n    [\\n      -76.9454025,\\n      38.988181166666664\\n    ],\\n    [\\n      -76.9453695,\\n      38.98850933333333\\n    ],\\n    [\\n      -76.94518766666667,\\n      38.9887485\\n    ],\\n    [\\n      -76.9446965,\\n      38.9886235\\n    ],\\n    [\\n      -76.944202,\\n      38.98865216666667\\n    ],\\n    [\\n      -76.94376466666667,\\n      38.9890515\\n    ],\\n    [\\n      -76.943237,\\n      38.98936116666667\\n    ],\\n    [\\n      -76.94251366666667,\\n      38.98938716666667\\n    ],\\n    [\\n      -76.94201966666667,\\n      38.989154\\n    ],\\n    [\\n      -76.94154416666666,\\n      38.98909233333333\\n    ],\\n    [\\n      -76.94144816666666,\\n      38.988639\\n    ],\\n    [\\n      -76.94159166666667,\\n      38.98789433333334\\n    ],\\n    [\\n      -76.9414205,\\n      38.98853416666667\\n    ],\\n    [\\n      -76.94140183333333,\\n      38.989425\\n    ],\\n    [\\n      -76.94133766666667,\\n      38.98997133333334\\n    ],\\n    [\\n      -76.94170066666666,\\n      38.990209666666665\\n    ],\\n    [\\n      -76.941905,\\n      38.9899985\\n    ],\\n    [\\n      -76.94191633333334,\\n      38.98957\\n    ],\\n    [\\n      -76.94168233333333,\\n      38.98928633333333\\n    ],\\n    [\\n      -76.9414125,\\n      38.98947733333333\\n    ],\\n    [\\n      -76.94160983333333,\\n      38.98998016666667\\n    ],\\n    [\\n      -76.94177083333334,\\n      38.989712\\n    ],\\n    [\\n      -76.94153116666666,\\n      38.989303166666666\\n    ],\\n    [\\n      -76.94134233333334,\\n      38.989587666666665\\n    ],\\n    [\\n      -76.9416585,\\n      38.989814333333335\\n    ],\\n    [\\n      -76.941737,\\n      38.99011333333333\\n    ],\\n    [\\n      -76.94168583333334,\\n      38.9898385\\n    ],\\n    [\\n      -76.94152166666667,\\n      38.989627666666664\\n    ],\\n    [\\n      -76.9416335,\\n      38.9894265\\n    ],\\n    [\\n      -76.94160833333333,\\n      38.990298\\n    ],\\n    [\\n      -76.94168766666667,\\n      38.990521\\n    ],\\n    [\\n      -76.9421955,\\n      38.99058133333333\\n    ],\\n    [\\n      -76.942843,\\n      38.990581166666665\\n    ],\\n    [\\n      -76.94362583333333,\\n      38.990572666666665\\n    ],\\n    [\\n      -76.94443283333334,\\n      38.99057083333334\\n    ],\\n    [\\n      -76.945238,\\n      38.990607833333335\\n    ],\\n    [\\n      -76.94598016666667,\\n      38.99081966666667\\n    ],\\n    [\\n      -76.94635566666666,\\n      38.9910145\\n    ],\\n    [\\n      -76.94648616666667,\\n      38.99124833333333\\n    ],\\n    [\\n      -76.94590866666667,\\n      38.99173316666667\\n    ],\\n    [\\n      -76.94561266666666,\\n      38.99212966666666\\n    ],\\n    [\\n      -76.945595,\\n      38.992660666666666\\n    ],\\n    [\\n      -76.94564766666667,\\n      38.99292716666667\\n    ],\\n    [\\n      -76.9454855,\\n      38.993093333333334\\n    ],\\n    [\\n      -76.94518416666666,\\n      38.993107\\n    ],\\n    [\\n      -76.94447683333334,\\n      38.993104333333335\\n    ],\\n    [\\n      -76.9439045,\\n      38.993072\\n    ],\\n    [\\n      -76.94326866666667,\\n      38.99307\\n    ],\\n    [\\n      -76.94281,\\n      38.993036333333336\\n    ],\\n    [\\n      -76.942002,\\n      38.9929205\\n    ],\\n    [\\n      -76.94178683333334,\\n      38.992558833333334\\n    ],\\n    [\\n      -76.94179133333333,\\n      38.992107\\n    ],\\n    [\\n      -76.94177516666667,\\n      38.9918595\\n    ],\\n    [\\n      -76.94176216666666,\\n      38.9916015\\n    ],\\n    [\\n      -76.9417345,\\n      38.991386\\n    ],\\n    [\\n      -76.94140366666667,\\n      38.99133783333333\\n    ],\\n    [\\n      -76.94103533333333,\\n      38.991298666666665\\n    ],\\n    [\\n      -76.94076083333333,\\n      38.9910725\\n    ],\\n    [\\n      -76.9407395,\\n      38.990777333333334\\n    ],\\n    [\\n      -76.94073783333333,\\n      38.99047016666667\\n    ],\\n    [\\n      -76.94077466666667,\\n      38.990132\\n    ],\\n    [\\n      -76.9407905,\\n      38.98975216666667\\n    ],\\n    [\\n      -76.9407925,\\n      38.98932583333333\\n    ],\\n    [\\n      -76.940788,\\n      38.989047666666664\\n    ],\\n    [\\n      -76.94070716666667,\\n      38.988801\\n    ],\\n    [\\n      -76.9406975,\\n      38.988465\\n    ],\\n    [\\n      -76.94063083333333,\\n      38.988010833333334\\n    ],\\n    [\\n      -76.9404885,\\n      38.9876425\\n    ],\\n    [\\n      -76.94019483333334,\\n      38.98712333333334\\n    ],\\n    [\\n      -76.9402235,\\n      38.986575333333334\\n    ],\\n    [\\n      -76.94059233333333,\\n      38.98657\\n    ],\\n    [\\n      -76.941057,\\n      38.9865655\\n    ],\\n    [\\n      -76.9415855,\\n      38.9865535\\n    ],\\n    [\\n      -76.94196683333334,\\n      38.98655883333333\\n    ],\\n    [\\n      -76.94219983333333,\\n      38.986666666666665\\n    ],\\n    [\\n      -76.942541,\\n      38.986854\\n    ],\\n    [\\n      -76.94270383333334,\\n      38.98690066666666\\n    ],\\n    [\\n      -76.94287066666666,\\n      38.98666033333333\\n    ],\\n    [\\n      -76.94322616666666,\\n      38.9865505\\n    ],\\n    [\\n      -76.94365233333333,\\n      38.98656483333333\\n    ],\\n    [\\n      -76.94410316666666,\\n      38.98652416666667\\n    ],\\n    [\\n      -76.94445816666666,\\n      38.9861435\\n    ],\\n    [\\n      -76.944486,\\n      38.98575833333334\\n    ],\\n    [\\n      -76.9443445,\\n      38.98558083333333\\n    ],\\n    [\\n      -76.94385483333333,\\n      38.985502\\n    ],\\n    [\\n      -76.943191,\\n      38.98547833333333\\n    ],\\n    [\\n      -76.9425365,\\n      38.985453166666666\\n    ],\\n    [\\n      -76.941837,\\n      38.985449\\n    ],\\n    [\\n      -76.9410995,\\n      38.9854645\\n    ],\\n    [\\n      -76.9405135,\\n      38.98546983333333\\n    ],\\n    [\\n      -76.94017766666667,\\n      38.9854345\\n    ],\\n    [\\n      -76.940162,\\n      38.984945833333335\\n    ],\\n    [\\n      -76.94014733333333,\\n      38.984641\\n    ],\\n    [\\n      -76.940628,\\n      38.98451466666667\\n    ],\\n    [\\n      -76.940909,\\n      38.98451566666667\\n    ],\\n    [\\n      -76.94123016666667,\\n      38.984505166666665\\n    ],\\n    [\\n      -76.94162366666667,\\n      38.984483\\n    ],\\n    [\\n      -76.94190533333334,\\n      38.9844755\\n    ],\\n    [\\n      -76.942182,\\n      38.984452833333336\\n    ],\\n    [\\n      -76.9423385,\\n      38.98427516666667\\n    ],\\n    [\\n      -76.94233316666667,\\n      38.98407983333333\\n    ],\\n    [\\n      -76.94232666666667,\\n      38.9839435\\n    ],\\n    [\\n      -76.942289,\\n      38.983792666666666\\n    ],\\n    [\\n      -76.94233083333333,\\n      38.983643\\n    ],\\n    [\\n      -76.94242233333334,\\n      38.983552333333336\\n    ],\\n    [\\n      -76.94242683333333,\\n      38.98340316666667\\n    ],\\n    [\\n      -76.94241416666667,\\n      38.983294666666666\\n    ],\\n    [\\n      -76.94244216666667,\\n      38.983193166666666\\n    ],\\n    [\\n      -76.9424725,\\n      38.983009\\n    ],\\n    [\\n      -76.94244916666666,\\n      38.9827975\\n    ],\\n    [\\n      -76.94242233333334,\\n      38.982652\\n    ],\\n    [\\n      -76.94244183333333,\\n      38.98243933333333\\n    ],\\n    [\\n      -76.942475,\\n      38.982204833333334\\n    ],\\n    [\\n      -76.94248133333333,\\n      38.98205133333333\\n    ],\\n    [\\n      -76.942495,\\n      38.98194133333333\\n    ],\\n    [\\n      -76.9425645,\\n      38.98180966666666\\n    ],\\n    [\\n      -76.9428185,\\n      38.981803\\n    ],\\n    [\\n      -76.9430225,\\n      38.981828166666666\\n    ],\\n    [\\n      -76.9432315,\\n      38.98182933333333\\n    ],\\n    [\\n      -76.94335966666667,\\n      38.981943\\n    ],\\n    [\\n      -76.94338416666666,\\n      38.98213633333334\\n    ],\\n    [\\n      -76.943451,\\n      38.98230516666667\\n    ],\\n    [\\n      -76.9437755,\\n      38.98234516666667\\n    ],\\n    [\\n      -76.94384616666666,\\n      38.98252766666667\\n    ],\\n    [\\n      -76.94360816666666,\\n      38.98256533333333\\n    ],\\n    [\\n      -76.94353733333334,\\n      38.982418833333334\\n    ],\\n    [\\n      -76.943797,\\n      38.982359333333335\\n    ],\\n    [\\n      -76.943857,\\n      38.98251\\n    ],\\n    [\\n      -76.9435945,\\n      38.98254083333333\\n    ],\\n    [\\n      -76.94356983333333,\\n      38.9823875\\n    ],\\n    [\\n      -76.94374766666667,\\n      38.982451\\n    ],\\n    [\\n      -76.94371983333333,\\n      38.98263766666667\\n    ],\\n    [\\n      -76.94355516666667,\\n      38.98259783333334\\n    ],\\n    [\\n      -76.94354133333333,\\n      38.98242033333333\\n    ],\\n    [\\n      -76.9437095,\\n      38.982410333333334\\n    ],\\n    [\\n      -76.94360166666667,\\n      38.982591666666664\\n    ],\\n    [\\n      -76.94354633333333,\\n      38.982475666666666\\n    ]\\n  ],\\n  "type": "MultiPoint"\\n}\',\n       \'{\\n  "coordinates": [\\n    [\\n      -76.93811566666666,\\n      38.9820835\\n    ],\\n    [\\n      -76.93826183333333,\\n      38.98215366666667\\n    ],\\n    [\\n      -76.93855833333333,\\n      38.982237833333336\\n    ],\\n    [\\n      -76.93880266666666,\\n      38.982306666666666\\n    ],\\n    [\\n      -76.93900033333334,\\n      38.982343166666666\\n    ],\\n    [\\n      -76.93920566666667,\\n      38.982413\\n    ],\\n    [\\n      -76.93935216666667,\\n      38.982450166666666\\n    ],\\n    [\\n      -76.93950516666666,\\n      38.982519\\n    ],\\n    [\\n      -76.93959666666666,\\n      38.982631\\n    ],\\n    [\\n      -76.93975283333333,\\n      38.9828475\\n    ],\\n    [\\n      -76.9399045,\\n      38.98299816666667\\n    ],\\n    [\\n      -76.94018266666667,\\n      38.982922\\n    ],\\n    [\\n      -76.94046583333333,\\n      38.98283133333334\\n    ],\\n    [\\n      -76.94076633333333,\\n      38.98290983333333\\n    ],\\n    [\\n      -76.940955,\\n      38.9830755\\n    ],\\n    [\\n      -76.94115716666667,\\n      38.983233\\n    ],\\n    [\\n      -76.9414785,\\n      38.9833955\\n    ],\\n    [\\n      -76.94170366666667,\\n      38.983528666666665\\n    ],\\n    [\\n      -76.94195966666666,\\n      38.98370966666667\\n    ],\\n    [\\n      -76.94216633333333,\\n      38.98384466666667\\n    ],\\n    [\\n      -76.94235416666666,\\n      38.98403116666667\\n    ],\\n    [\\n      -76.94256233333333,\\n      38.984220666666666\\n    ],\\n    [\\n      -76.94270366666667,\\n      38.984397333333334\\n    ],\\n    [\\n      -76.9426445,\\n      38.9845965\\n    ],\\n    [\\n      -76.94262533333334,\\n      38.984759833333335\\n    ],\\n    [\\n      -76.942626,\\n      38.98495033333333\\n    ],\\n    [\\n      -76.942646,\\n      38.98513416666667\\n    ],\\n    [\\n      -76.94272166666667,\\n      38.985287166666666\\n    ],\\n    [\\n      -76.94283366666667,\\n      38.985417\\n    ],\\n    [\\n      -76.94308516666666,\\n      38.985630666666665\\n    ],\\n    [\\n      -76.9432275,\\n      38.985928\\n    ],\\n    [\\n      -76.943257,\\n      38.986101\\n    ]\\n  ],\\n  "type": "MultiPoint"\\n}\',\n       \'{\\n  "coordinates": [\\n    [\\n      -76.94269666666666,\\n      38.98548483333333\\n    ],\\n    [\\n      -76.9429295,\\n      38.9854895\\n    ],\\n    [\\n      -76.943135,\\n      38.98548383333333\\n    ],\\n    [\\n      -76.9433015,\\n      38.98548316666667\\n    ],\\n    [\\n      -76.94318166666666,\\n      38.9853795\\n    ],\\n    [\\n      -76.94298333333333,\\n      38.9853065\\n    ]\\n  ],\\n  "type": "MultiPoint"\\n}\',\n       \'{\\n  "coordinates": [\\n    [\\n      -76.93853383333334,\\n      38.98101616666667\\n    ],\\n    [\\n      -76.93870183333334,\\n      38.98103066666667\\n    ],\\n    [\\n      -76.9388775,\\n      38.98105916666667\\n    ],\\n    [\\n      -76.93908416666666,\\n      38.981115\\n    ],\\n    [\\n      -76.93906366666667,\\n      38.98126383333334\\n    ],\\n    [\\n      -76.93909766666667,\\n      38.981416333333335\\n    ],\\n    [\\n      -76.93925466666667,\\n      38.981449\\n    ],\\n    [\\n      -76.939106,\\n      38.9811825\\n    ],\\n    [\\n      -76.9390145,\\n      38.981074\\n    ],\\n    [\\n      -76.938937,\\n      38.9812085\\n    ],\\n    [\\n      -76.938961,\\n      38.9815285\\n    ],\\n    [\\n      -76.93898533333333,\\n      38.981713666666664\\n    ],\\n    [\\n      -76.938902,\\n      38.98179283333333\\n    ],\\n    [\\n      -76.93865966666667,\\n      38.98185516666667\\n    ],\\n    [\\n      -76.93863816666666,\\n      38.98202833333333\\n    ],\\n    [\\n      -76.93875,\\n      38.982194666666665\\n    ],\\n    [\\n      -76.93885416666667,\\n      38.98228133333333\\n    ],\\n    [\\n      -76.938994,\\n      38.98232616666667\\n    ],\\n    [\\n      -76.939125,\\n      38.982481166666666\\n    ],\\n    [\\n      -76.93928616666666,\\n      38.98258666666667\\n    ],\\n    [\\n      -76.93952383333334,\\n      38.982705333333335\\n    ],\\n    [\\n      -76.93978416666667,\\n      38.982966833333336\\n    ],\\n    [\\n      -76.93990633333334,\\n      38.983241666666665\\n    ],\\n    [\\n      -76.93995866666667,\\n      38.98351866666667\\n    ],\\n    [\\n      -76.94000916666667,\\n      38.9838775\\n    ],\\n    [\\n      -76.940031,\\n      38.984179\\n    ],\\n    [\\n      -76.94000983333333,\\n      38.9844995\\n    ],\\n    [\\n      -76.94002633333334,\\n      38.984881\\n    ],\\n    [\\n      -76.94003933333333,\\n      38.985117\\n    ],\\n    [\\n      -76.94005516666667,\\n      38.985406833333336\\n    ],\\n    [\\n      -76.940051,\\n      38.985600833333336\\n    ],\\n    [\\n      -76.94006633333333,\\n      38.98586133333333\\n    ],\\n    [\\n      -76.9400565,\\n      38.986245333333336\\n    ],\\n    [\\n      -76.94010683333333,\\n      38.986413166666665\\n    ],\\n    [\\n      -76.940202,\\n      38.98654616666666\\n    ],\\n    [\\n      -76.94045816666667,\\n      38.986576166666666\\n    ],\\n    [\\n      -76.94075050000001,\\n      38.98656783333333\\n    ],\\n    [\\n      -76.94111566666666,\\n      38.98658016666667\\n    ],\\n    [\\n      -76.94154966666666,\\n      38.98657883333333\\n    ],\\n    [\\n      -76.94186266666667,\\n      38.98659\\n    ],\\n    [\\n      -76.94206366666667,\\n      38.986624666666664\\n    ],\\n    [\\n      -76.94221916666666,\\n      38.986581\\n    ]\\n  ],\\n  "type": "MultiPoint"\\n}\',\n       \'{\\n  "coordinates": [\\n    [\\n      -76.93679183333333,\\n      38.988329\\n    ],\\n    [\\n      -76.93699216666667,\\n      38.98833916666667\\n    ],\\n    [\\n      -76.93718566666666,\\n      38.988326666666666\\n    ],\\n    [\\n      -76.93742816666666,\\n      38.9882855\\n    ],\\n    [\\n      -76.9376695,\\n      38.98821266666667\\n    ],\\n    [\\n      -76.93789416666667,\\n      38.9881345\\n    ],\\n    [\\n      -76.9381595,\\n      38.9880425\\n    ],\\n    [\\n      -76.9383475,\\n      38.98798566666667\\n    ],\\n    [\\n      -76.9384865,\\n      38.98797016666666\\n    ],\\n    [\\n      -76.938644,\\n      38.98796033333333\\n    ],\\n    [\\n      -76.93888016666666,\\n      38.987893666666665\\n    ],\\n    [\\n      -76.93911983333334,\\n      38.98782933333333\\n    ],\\n    [\\n      -76.93934566666667,\\n      38.987756\\n    ],\\n    [\\n      -76.939568,\\n      38.987647333333335\\n    ],\\n    [\\n      -76.93971733333333,\\n      38.987543\\n    ],\\n    [\\n      -76.9398685,\\n      38.987440166666666\\n    ],\\n    [\\n      -76.939956,\\n      38.98735083333333\\n    ],\\n    [\\n      -76.94002283333333,\\n      38.98719416666667\\n    ],\\n    [\\n      -76.93999083333334,\\n      38.98703183333333\\n    ]\\n  ],\\n  "type": "MultiPoint"\\n}\',\n       \'{\\n  "coordinates": [\\n    [\\n      -76.934581,\\n      38.983489166666665\\n    ],\\n    [\\n      -76.9348115,\\n      38.983454\\n    ],\\n    [\\n      -76.9351365,\\n      38.983442\\n    ],\\n    [\\n      -76.93540633333333,\\n      38.983446\\n    ],\\n    [\\n      -76.93562833333333,\\n      38.98345616666667\\n    ],\\n    [\\n      -76.9357815,\\n      38.983422\\n    ],\\n    [\\n      -76.93578533333333,\\n      38.98327466666667\\n    ],\\n    [\\n      -76.93585666666667,\\n      38.9830555\\n    ],\\n    [\\n      -76.93561266666667,\\n      38.9830065\\n    ],\\n    [\\n      -76.9353425,\\n      38.9830105\\n    ],\\n    [\\n      -76.93502216666667,\\n      38.983019\\n    ],\\n    [\\n      -76.93479433333333,\\n      38.983034833333335\\n    ],\\n    [\\n      -76.93453433333333,\\n      38.98303216666667\\n    ],\\n    [\\n      -76.93428466666667,\\n      38.98308216666667\\n    ],\\n    [\\n      -76.93426833333334,\\n      38.98298283333333\\n    ],\\n    [\\n      -76.93430616666667,\\n      38.982861666666665\\n    ],\\n    [\\n      -76.93423133333333,\\n      38.9830345\\n    ],\\n    [\\n      -76.9342645,\\n      38.98317266666667\\n    ],\\n    [\\n      -76.93439966666666,\\n      38.98330166666667\\n    ],\\n    [\\n      -76.93450083333333,\\n      38.9834415\\n    ]\\n  ],\\n  "type": "MultiPoint"\\n}\',\n       \'{\\n  "coordinates": [\\n    [\\n      -76.9343025,\\n      38.99064083333333\\n    ],\\n    [\\n      -76.93446133333333,\\n      38.99055383333334\\n    ],\\n    [\\n      -76.93470533333333,\\n      38.990479\\n    ],\\n    [\\n      -76.934919,\\n      38.990398\\n    ],\\n    [\\n      -76.935111,\\n      38.9903235\\n    ],\\n    [\\n      -76.93536633333333,\\n      38.99024516666667\\n    ],\\n    [\\n      -76.93558633333333,\\n      38.990229166666666\\n    ],\\n    [\\n      -76.935859,\\n      38.990214\\n    ],\\n    [\\n      -76.9362175,\\n      38.99020266666667\\n    ],\\n    [\\n      -76.9364375,\\n      38.990183333333334\\n    ],\\n    [\\n      -76.93663716666667,\\n      38.9902575\\n    ],\\n    [\\n      -76.93695316666667,\\n      38.990265666666666\\n    ],\\n    [\\n      -76.93728733333333,\\n      38.9902455\\n    ],\\n    [\\n      -76.93769516666667,\\n      38.9902415\\n    ],\\n    [\\n      -76.93817066666666,\\n      38.990226\\n    ],\\n    [\\n      -76.93855816666667,\\n      38.99022566666667\\n    ],\\n    [\\n      -76.93897016666666,\\n      38.990233833333335\\n    ],\\n    [\\n      -76.93942783333334,\\n      38.990239\\n    ],\\n    [\\n      -76.93977433333333,\\n      38.990232666666664\\n    ],\\n    [\\n      -76.94012733333334,\\n      38.990240166666666\\n    ],\\n    [\\n      -76.94049533333333,\\n      38.99023666666667\\n    ],\\n    [\\n      -76.94075383333333,\\n      38.990228\\n    ],\\n    [\\n      -76.94108083333333,\\n      38.990303\\n    ],\\n    [\\n      -76.94133066666667,\\n      38.9904335\\n    ],\\n    [\\n      -76.9416605,\\n      38.990601166666664\\n    ],\\n    [\\n      -76.94200566666666,\\n      38.99064933333333\\n    ],\\n    [\\n      -76.94230416666667,\\n      38.9906465\\n    ],\\n    [\\n      -76.94270083333333,\\n      38.990637\\n    ],\\n    [\\n      -76.943098,\\n      38.99063316666667\\n    ],\\n    [\\n      -76.94350833333333,\\n      38.990624833333335\\n    ],\\n    [\\n      -76.943978,\\n      38.99060766666667\\n    ],\\n    [\\n      -76.94438,\\n      38.990604\\n    ],\\n    [\\n      -76.94479633333333,\\n      38.990603666666665\\n    ],\\n    [\\n      -76.94518566666666,\\n      38.99062283333333\\n    ],\\n    [\\n      -76.94562016666667,\\n      38.99072783333333\\n    ],\\n    [\\n      -76.94595366666667,\\n      38.99083516666666\\n    ],\\n    [\\n      -76.94622233333334,\\n      38.990942833333335\\n    ],\\n    [\\n      -76.94646366666667,\\n      38.9911505\\n    ],\\n    [\\n      -76.94629366666666,\\n      38.99139516666666\\n    ],\\n    [\\n      -76.94603633333334,\\n      38.99163183333334\\n    ],\\n    [\\n      -76.945741,\\n      38.9919005\\n    ],\\n    [\\n      -76.94562683333334,\\n      38.992198\\n    ],\\n    [\\n      -76.945606,\\n      38.992489166666665\\n    ],\\n    [\\n      -76.94563016666666,\\n      38.992724\\n    ],\\n    [\\n      -76.94562016666667,\\n      38.99282616666667\\n    ]\\n  ],\\n  "type": "MultiPoint"\\n}\',\n       ...\n       \'{\\n  "coordinates": [\\n    [\\n      -76.935309,\\n      38.986512\\n    ],\\n    [\\n      -76.9354705,\\n      38.986590666666665\\n    ],\\n    [\\n      -76.9356885,\\n      38.98666866666667\\n    ],\\n    [\\n      -76.93583733333334,\\n      38.98672883333333\\n    ],\\n    [\\n      -76.936031,\\n      38.986799\\n    ],\\n    [\\n      -76.93621433333334,\\n      38.986887833333334\\n    ],\\n    [\\n      -76.93637966666667,\\n      38.986947\\n    ],\\n    [\\n      -76.936546,\\n      38.9867835\\n    ],\\n    [\\n      -76.93661016666667,\\n      38.98667433333333\\n    ],\\n    [\\n      -76.93669733333333,\\n      38.9864855\\n    ],\\n    [\\n      -76.93677183333334,\\n      38.9863575\\n    ],\\n    [\\n      -76.9368625,\\n      38.986199166666665\\n    ],\\n    [\\n      -76.93694933333333,\\n      38.98605583333333\\n    ],\\n    [\\n      -76.93703783333333,\\n      38.985910833333335\\n    ],\\n    [\\n      -76.937098,\\n      38.9857845\\n    ],\\n    [\\n      -76.9371825,\\n      38.985635333333335\\n    ],\\n    [\\n      -76.937259,\\n      38.98550016666667\\n    ],\\n    [\\n      -76.93731633333333,\\n      38.985355166666665\\n    ],\\n    [\\n      -76.93739933333333,\\n      38.98517983333333\\n    ],\\n    [\\n      -76.93750833333333,\\n      38.98501133333333\\n    ],\\n    [\\n      -76.93758116666666,\\n      38.98483266666667\\n    ],\\n    [\\n      -76.93766533333333,\\n      38.984620666666665\\n    ],\\n    [\\n      -76.93773416666667,\\n      38.9843925\\n    ],\\n    [\\n      -76.93781,\\n      38.98414866666667\\n    ],\\n    [\\n      -76.9378755,\\n      38.983907333333335\\n    ],\\n    [\\n      -76.9379305,\\n      38.983655166666665\\n    ],\\n    [\\n      -76.937977,\\n      38.98345183333333\\n    ],\\n    [\\n      -76.9380015,\\n      38.98330333333333\\n    ],\\n    [\\n      -76.93803866666667,\\n      38.98307483333333\\n    ],\\n    [\\n      -76.93804883333334,\\n      38.982936333333335\\n    ],\\n    [\\n      -76.93807566666666,\\n      38.982649333333335\\n    ],\\n    [\\n      -76.938074,\\n      38.982469\\n    ],\\n    [\\n      -76.93804683333333,\\n      38.98230616666667\\n    ],\\n    [\\n      -76.93804116666666,\\n      38.982173833333334\\n    ],\\n    [\\n      -76.93806666666667,\\n      38.9820095\\n    ],\\n    [\\n      -76.93808983333334,\\n      38.981893166666666\\n    ],\\n    [\\n      -76.93809833333333,\\n      38.9817355\\n    ],\\n    [\\n      -76.9381305,\\n      38.981634666666665\\n    ],\\n    [\\n      -76.93814466666667,\\n      38.981519666666664\\n    ]\\n  ],\\n  "type": "MultiPoint"\\n}\',\n       \'{\\n  "coordinates": [\\n    [\\n      -76.94084483333333,\\n      38.98991983333333\\n    ],\\n    [\\n      -76.94081616666666,\\n      38.989753\\n    ],\\n    [\\n      -76.9408025,\\n      38.98941816666667\\n    ],\\n    [\\n      -76.9407935,\\n      38.98919016666667\\n    ],\\n    [\\n      -76.94078433333334,\\n      38.989032\\n    ],\\n    [\\n      -76.94071916666667,\\n      38.98881333333333\\n    ],\\n    [\\n      -76.94071533333333,\\n      38.988518\\n    ],\\n    [\\n      -76.94071316666667,\\n      38.98820083333333\\n    ],\\n    [\\n      -76.940722,\\n      38.9880885\\n    ],\\n    [\\n      -76.9408205,\\n      38.98782533333333\\n    ],\\n    [\\n      -76.94092466666666,\\n      38.987667333333334\\n    ],\\n    [\\n      -76.94083816666667,\\n      38.98753216666667\\n    ],\\n    [\\n      -76.94053483333333,\\n      38.98749016666667\\n    ],\\n    [\\n      -76.94029383333333,\\n      38.98720466666666\\n    ],\\n    [\\n      -76.94027166666666,\\n      38.986984666666665\\n    ],\\n    [\\n      -76.94014433333334,\\n      38.98693933333333\\n    ],\\n    [\\n      -76.94005533333333,\\n      38.987193166666664\\n    ],\\n    [\\n      -76.93987233333333,\\n      38.98747483333333\\n    ],\\n    [\\n      -76.93948833333333,\\n      38.987666\\n    ],\\n    [\\n      -76.93908933333333,\\n      38.98775916666666\\n    ],\\n    [\\n      -76.93862366666667,\\n      38.987894\\n    ],\\n    [\\n      -76.9382585,\\n      38.988017166666666\\n    ],\\n    [\\n      -76.937893,\\n      38.9881555\\n    ],\\n    [\\n      -76.93750766666666,\\n      38.988271\\n    ],\\n    [\\n      -76.93702866666666,\\n      38.9883645\\n    ],\\n    [\\n      -76.93671966666666,\\n      38.98836933333333\\n    ],\\n    [\\n      -76.936447,\\n      38.98838\\n    ],\\n    [\\n      -76.9359915,\\n      38.98833166666667\\n    ],\\n    [\\n      -76.93577716666667,\\n      38.988097833333335\\n    ],\\n    [\\n      -76.93580783333333,\\n      38.9879675\\n    ],\\n    [\\n      -76.93590483333334,\\n      38.987772666666665\\n    ],\\n    [\\n      -76.936022,\\n      38.9875575\\n    ],\\n    [\\n      -76.93608766666667,\\n      38.987440666666664\\n    ],\\n    [\\n      -76.93618316666667,\\n      38.987243666666664\\n    ],\\n    [\\n      -76.9362985,\\n      38.987083\\n    ],\\n    [\\n      -76.9363865,\\n      38.98693883333333\\n    ],\\n    [\\n      -76.936503,\\n      38.986753666666665\\n    ],\\n    [\\n      -76.93667366666666,\\n      38.98645583333333\\n    ],\\n    [\\n      -76.93679766666666,\\n      38.986242\\n    ],\\n    [\\n      -76.9369295,\\n      38.98601716666667\\n    ],\\n    [\\n      -76.93706383333334,\\n      38.985782666666665\\n    ],\\n    [\\n      -76.937189,\\n      38.98555283333334\\n    ],\\n    [\\n      -76.93732116666666,\\n      38.985248\\n    ],\\n    [\\n      -76.93748583333333,\\n      38.9849465\\n    ],\\n    [\\n      -76.93762583333333,\\n      38.9845685\\n    ],\\n    [\\n      -76.93772233333334,\\n      38.984258\\n    ],\\n    [\\n      -76.93781066666666,\\n      38.983950166666666\\n    ],\\n    [\\n      -76.937889,\\n      38.983596666666664\\n    ],\\n    [\\n      -76.93795383333334,\\n      38.983271333333334\\n    ],\\n    [\\n      -76.93800016666667,\\n      38.98295366666667\\n    ],\\n    [\\n      -76.93803533333333,\\n      38.98268516666667\\n    ],\\n    [\\n      -76.93806483333333,\\n      38.982454\\n    ]\\n  ],\\n  "type": "MultiPoint"\\n}\',\n       \'{\\n  "coordinates": [\\n    [\\n      -76.938891,\\n      38.980524833333334\\n    ],\\n    [\\n      -76.93884766666666,\\n      38.980323166666665\\n    ],\\n    [\\n      -76.93886,\\n      38.98021933333333\\n    ],\\n    [\\n      -76.93880816666666,\\n      38.98012\\n    ],\\n    [\\n      -76.938679,\\n      38.980093833333335\\n    ],\\n    [\\n      -76.9384955,\\n      38.9801145\\n    ],\\n    [\\n      -76.938208,\\n      38.980129\\n    ],\\n    [\\n      -76.93808933333334,\\n      38.9803045\\n    ],\\n    [\\n      -76.93808733333333,\\n      38.98041116666667\\n    ],\\n    [\\n      -76.9380765,\\n      38.9805345\\n    ],\\n    [\\n      -76.93807383333333,\\n      38.980652166666665\\n    ],\\n    [\\n      -76.93807266666667,\\n      38.980777333333336\\n    ],\\n    [\\n      -76.938056,\\n      38.98088633333333\\n    ],\\n    [\\n      -76.93805216666667,\\n      38.98102633333333\\n    ],\\n    [\\n      -76.93805966666666,\\n      38.981168833333335\\n    ],\\n    [\\n      -76.93805066666667,\\n      38.981328\\n    ],\\n    [\\n      -76.93811583333333,\\n      38.981537\\n    ],\\n    [\\n      -76.93810033333334,\\n      38.981647\\n    ],\\n    [\\n      -76.93807983333333,\\n      38.981822333333334\\n    ],\\n    [\\n      -76.93808583333333,\\n      38.981990333333336\\n    ],\\n    [\\n      -76.93819316666666,\\n      38.982109\\n    ],\\n    [\\n      -76.93841883333333,\\n      38.982184333333336\\n    ],\\n    [\\n      -76.9385645,\\n      38.98222333333333\\n    ],\\n    [\\n      -76.93887816666667,\\n      38.982315\\n    ],\\n    [\\n      -76.9391015,\\n      38.98235833333333\\n    ],\\n    [\\n      -76.93933766666666,\\n      38.98242833333333\\n    ],\\n    [\\n      -76.93951516666667,\\n      38.9824935\\n    ],\\n    [\\n      -76.939663,\\n      38.982630666666665\\n    ],\\n    [\\n      -76.93981133333334,\\n      38.982814166666664\\n    ],\\n    [\\n      -76.939917,\\n      38.98297916666667\\n    ],\\n    [\\n      -76.9399635,\\n      38.983125666666666\\n    ],\\n    [\\n      -76.939984,\\n      38.98327616666667\\n    ],\\n    [\\n      -76.94003366666666,\\n      38.9834185\\n    ],\\n    [\\n      -76.94005516666667,\\n      38.983532833333335\\n    ],\\n    [\\n      -76.94008833333334,\\n      38.983771\\n    ],\\n    [\\n      -76.94013283333334,\\n      38.98394283333333\\n    ],\\n    [\\n      -76.94016433333333,\\n      38.984049166666665\\n    ],\\n    [\\n      -76.9402525,\\n      38.98395383333333\\n    ],\\n    [\\n      -76.94035316666667,\\n      38.983885\\n    ]\\n  ],\\n  "type": "MultiPoint"\\n}\',\n       \'{\\n  "coordinates": [\\n    [\\n      -76.94980733333334,\\n      38.9921135\\n    ],\\n    [\\n      -76.94964966666667,\\n      38.99203116666666\\n    ],\\n    [\\n      -76.94948483333333,\\n      38.991923166666666\\n    ],\\n    [\\n      -76.9493855,\\n      38.99179916666667\\n    ],\\n    [\\n      -76.94925883333333,\\n      38.99170266666667\\n    ],\\n    [\\n      -76.9492195,\\n      38.9915225\\n    ],\\n    [\\n      -76.94927566666667,\\n      38.99141383333333\\n    ],\\n    [\\n      -76.94944416666667,\\n      38.9913235\\n    ],\\n    [\\n      -76.94941033333333,\\n      38.99108716666667\\n    ],\\n    [\\n      -76.94941833333333,\\n      38.9907885\\n    ],\\n    [\\n      -76.9494225,\\n      38.99047233333334\\n    ],\\n    [\\n      -76.94942166666667,\\n      38.9901555\\n    ],\\n    [\\n      -76.9494155,\\n      38.989835666666664\\n    ],\\n    [\\n      -76.9494245,\\n      38.989519666666666\\n    ],\\n    [\\n      -76.94957683333334,\\n      38.989246666666666\\n    ],\\n    [\\n      -76.94962333333334,\\n      38.98893483333333\\n    ],\\n    [\\n      -76.94961833333333,\\n      38.9886105\\n    ],\\n    [\\n      -76.9496275,\\n      38.98828483333333\\n    ],\\n    [\\n      -76.94963216666666,\\n      38.98796016666667\\n    ],\\n    [\\n      -76.94963966666667,\\n      38.987635833333336\\n    ],\\n    [\\n      -76.94964583333334,\\n      38.987309\\n    ],\\n    [\\n      -76.94964566666667,\\n      38.986982\\n    ],\\n    [\\n      -76.94964866666666,\\n      38.986658166666665\\n    ],\\n    [\\n      -76.94965733333333,\\n      38.9863395\\n    ],\\n    [\\n      -76.9496605,\\n      38.98600966666667\\n    ],\\n    [\\n      -76.94965616666667,\\n      38.985688333333336\\n    ],\\n    [\\n      -76.94965,\\n      38.9854165\\n    ],\\n    [\\n      -76.94952933333333,\\n      38.98521483333333\\n    ],\\n    [\\n      -76.9492205,\\n      38.985205\\n    ],\\n    [\\n      -76.94903133333334,\\n      38.98515\\n    ],\\n    [\\n      -76.94904066666666,\\n      38.984957666666666\\n    ],\\n    [\\n      -76.94895733333334,\\n      38.984727666666664\\n    ],\\n    [\\n      -76.94876516666666,\\n      38.984504\\n    ],\\n    [\\n      -76.94864966666667,\\n      38.98418233333334\\n    ],\\n    [\\n      -76.94852766666666,\\n      38.983857666666665\\n    ],\\n    [\\n      -76.9483905,\\n      38.98352833333333\\n    ],\\n    [\\n      -76.94826016666667,\\n      38.983215666666666\\n    ],\\n    [\\n      -76.94814416666667,\\n      38.982913\\n    ],\\n    [\\n      -76.94791966666666,\\n      38.9826435\\n    ],\\n    [\\n      -76.94769716666667,\\n      38.982384333333336\\n    ],\\n    [\\n      -76.94735166666666,\\n      38.982293166666665\\n    ],\\n    [\\n      -76.94697233333333,\\n      38.98219616666667\\n    ],\\n    [\\n      -76.946751,\\n      38.981871166666664\\n    ],\\n    [\\n      -76.94656916666666,\\n      38.98156083333333\\n    ],\\n    [\\n      -76.94622133333333,\\n      38.981402833333334\\n    ],\\n    [\\n      -76.94580683333334,\\n      38.981389\\n    ],\\n    [\\n      -76.94544466666666,\\n      38.98137766666667\\n    ],\\n    [\\n      -76.9451415,\\n      38.98135083333333\\n    ],\\n    [\\n      -76.94489216666666,\\n      38.981392666666665\\n    ],\\n    [\\n      -76.9448735,\\n      38.98163483333333\\n    ],\\n    [\\n      -76.94486016666667,\\n      38.981753166666664\\n    ],\\n    [\\n      -76.94474583333333,\\n      38.981816\\n    ]\\n  ],\\n  "type": "MultiPoint"\\n}\',\n       \'{\\n  "coordinates": [\\n    [\\n      -76.93533283333333,\\n      38.97906433333333\\n    ],\\n    [\\n      -76.93514966666666,\\n      38.979032\\n    ],\\n    [\\n      -76.9348045,\\n      38.97892683333333\\n    ],\\n    [\\n      -76.93447233333333,\\n      38.978851\\n    ],\\n    [\\n      -76.93412983333333,\\n      38.978750833333336\\n    ],\\n    [\\n      -76.93371566666667,\\n      38.978760666666666\\n    ],\\n    [\\n      -76.933316,\\n      38.978805\\n    ],\\n    [\\n      -76.93290916666666,\\n      38.978857166666664\\n    ],\\n    [\\n      -76.93252683333333,\\n      38.97884916666667\\n    ],\\n    [\\n      -76.93240833333333,\\n      38.97864833333333\\n    ],\\n    [\\n      -76.9325175,\\n      38.97836516666667\\n    ],\\n    [\\n      -76.93259916666666,\\n      38.9780815\\n    ],\\n    [\\n      -76.93269483333333,\\n      38.977763833333334\\n    ],\\n    [\\n      -76.9328115,\\n      38.97745783333333\\n    ],\\n    [\\n      -76.93290383333333,\\n      38.977163166666664\\n    ],\\n    [\\n      -76.93293033333333,\\n      38.97694933333333\\n    ],\\n    [\\n      -76.9326015,\\n      38.97687416666667\\n    ],\\n    [\\n      -76.9322815,\\n      38.976823333333336\\n    ],\\n    [\\n      -76.93188833333333,\\n      38.976746\\n    ],\\n    [\\n      -76.93149416666667,\\n      38.97666816666667\\n    ],\\n    [\\n      -76.93112316666667,\\n      38.97661383333333\\n    ],\\n    [\\n      -76.93075683333333,\\n      38.9765265\\n    ],\\n    [\\n      -76.930382,\\n      38.9764515\\n    ],\\n    [\\n      -76.93001533333333,\\n      38.97638533333333\\n    ],\\n    [\\n      -76.92974433333333,\\n      38.976315666666665\\n    ],\\n    [\\n      -76.92969933333333,\\n      38.976138\\n    ],\\n    [\\n      -76.92978616666667,\\n      38.97584833333333\\n    ],\\n    [\\n      -76.9299905,\\n      38.97564116666667\\n    ],\\n    [\\n      -76.930235,\\n      38.9756055\\n    ]\\n  ],\\n  "type": "MultiPoint"\\n}\',\n       \'{\\n  "coordinates": [\\n    [\\n      -76.9439365,\\n      38.98811\\n    ],\\n    [\\n      -76.94407783333334,\\n      38.987963\\n    ],\\n    [\\n      -76.94413783333333,\\n      38.98783733333333\\n    ],\\n    [\\n      -76.94408366666667,\\n      38.98772016666667\\n    ],\\n    [\\n      -76.94390683333333,\\n      38.98765816666667\\n    ],\\n    [\\n      -76.94359016666667,\\n      38.9876665\\n    ],\\n    [\\n      -76.94320616666667,\\n      38.987676666666665\\n    ],\\n    [\\n      -76.9428165,\\n      38.9876905\\n    ],\\n    [\\n      -76.94253133333333,\\n      38.98769683333333\\n    ],\\n    [\\n      -76.94223733333334,\\n      38.9876995\\n    ],\\n    [\\n      -76.9419505,\\n      38.98769866666667\\n    ],\\n    [\\n      -76.94162983333334,\\n      38.98770116666667\\n    ],\\n    [\\n      -76.94136433333334,\\n      38.987701666666666\\n    ],\\n    [\\n      -76.94115433333333,\\n      38.98770183333333\\n    ],\\n    [\\n      -76.94093333333333,\\n      38.98774083333333\\n    ],\\n    [\\n      -76.94077116666666,\\n      38.987885166666665\\n    ],\\n    [\\n      -76.94070416666666,\\n      38.988140333333334\\n    ],\\n    [\\n      -76.94070583333334,\\n      38.988405166666666\\n    ],\\n    [\\n      -76.9407115,\\n      38.98871166666667\\n    ],\\n    [\\n      -76.9406865,\\n      38.98897516666667\\n    ],\\n    [\\n      -76.940651,\\n      38.98929316666667\\n    ],\\n    [\\n      -76.94064533333334,\\n      38.9896805\\n    ],\\n    [\\n      -76.94067566666666,\\n      38.989997333333335\\n    ],\\n    [\\n      -76.94067133333333,\\n      38.99031466666667\\n    ],\\n    [\\n      -76.940666,\\n      38.99063833333334\\n    ],\\n    [\\n      -76.94066533333333,\\n      38.991017\\n    ],\\n    [\\n      -76.94069133333333,\\n      38.99130266666667\\n    ],\\n    [\\n      -76.94051033333334,\\n      38.99138283333333\\n    ],\\n    [\\n      -76.940086,\\n      38.991372166666665\\n    ],\\n    [\\n      -76.9398085,\\n      38.99134633333333\\n    ],\\n    [\\n      -76.93934116666667,\\n      38.991310166666665\\n    ],\\n    [\\n      -76.9388675,\\n      38.99132\\n    ],\\n    [\\n      -76.93842966666666,\\n      38.99137066666667\\n    ],\\n    [\\n      -76.93800783333333,\\n      38.991373333333335\\n    ],\\n    [\\n      -76.93764716666666,\\n      38.991451833333336\\n    ],\\n    [\\n      -76.93751583333334,\\n      38.991829333333335\\n    ],\\n    [\\n      -76.937499,\\n      38.99215266666667\\n    ],\\n    [\\n      -76.93748666666667,\\n      38.99245533333333\\n    ],\\n    [\\n      -76.9373555,\\n      38.99276316666667\\n    ],\\n    [\\n      -76.93712516666666,\\n      38.992948\\n    ],\\n    [\\n      -76.9367865,\\n      38.99289366666667\\n    ],\\n    [\\n      -76.9363505,\\n      38.9927795\\n    ],\\n    [\\n      -76.93597466666667,\\n      38.992731\\n    ],\\n    [\\n      -76.9356685,\\n      38.992837\\n    ],\\n    [\\n      -76.93547233333334,\\n      38.9927775\\n    ],\\n    [\\n      -76.9352115,\\n      38.99268766666667\\n    ],\\n    [\\n      -76.935068,\\n      38.9925525\\n    ],\\n    [\\n      -76.93498966666667,\\n      38.99235433333333\\n    ],\\n    [\\n      -76.934714,\\n      38.99226133333333\\n    ],\\n    [\\n      -76.93440083333333,\\n      38.99216\\n    ],\\n    [\\n      -76.93410433333334,\\n      38.9920605\\n    ]\\n  ],\\n  "type": "MultiPoint"\\n}\',\n       \'{\\n  "coordinates": [\\n    [\\n      -76.93591716666667,\\n      38.98638866666667\\n    ],\\n    [\\n      -76.93608516666667,\\n      38.986551\\n    ],\\n    [\\n      -76.93606916666667,\\n      38.9866975\\n    ],\\n    [\\n      -76.93597916666667,\\n      38.9868115\\n    ],\\n    [\\n      -76.93576983333334,\\n      38.98703883333334\\n    ],\\n    [\\n      -76.93560833333333,\\n      38.98723916666667\\n    ],\\n    [\\n      -76.93547483333333,\\n      38.98738916666667\\n    ],\\n    [\\n      -76.93535133333333,\\n      38.9875635\\n    ],\\n    [\\n      -76.93522866666666,\\n      38.987824833333335\\n    ],\\n    [\\n      -76.93510183333333,\\n      38.98803816666667\\n    ],\\n    [\\n      -76.93499933333334,\\n      38.98818083333333\\n    ],\\n    [\\n      -76.93510466666666,\\n      38.98831833333333\\n    ],\\n    [\\n      -76.93533133333334,\\n      38.988359833333334\\n    ],\\n    [\\n      -76.93545466666667,\\n      38.988399333333334\\n    ],\\n    [\\n      -76.93538933333333,\\n      38.988588166666666\\n    ],\\n    [\\n      -76.9352525,\\n      38.98879216666667\\n    ],\\n    [\\n      -76.935083,\\n      38.9890245\\n    ],\\n    [\\n      -76.93491116666667,\\n      38.98926616666667\\n    ],\\n    [\\n      -76.9347615,\\n      38.9894955\\n    ],\\n    [\\n      -76.93463216666666,\\n      38.98970816666667\\n    ],\\n    [\\n      -76.93452,\\n      38.98989433333333\\n    ],\\n    [\\n      -76.9344885,\\n      38.990050333333336\\n    ],\\n    [\\n      -76.93440783333334,\\n      38.99014733333333\\n    ],\\n    [\\n      -76.93428166666666,\\n      38.9903635\\n    ],\\n    [\\n      -76.93418483333333,\\n      38.99053116666666\\n    ],\\n    [\\n      -76.93405266666667,\\n      38.990845\\n    ],\\n    [\\n      -76.93397366666667,\\n      38.991075333333335\\n    ],\\n    [\\n      -76.93390083333334,\\n      38.99140216666667\\n    ],\\n    [\\n      -76.9338665,\\n      38.991639666666664\\n    ],\\n    [\\n      -76.93366833333333,\\n      38.991890833333336\\n    ],\\n    [\\n      -76.93355966666667,\\n      38.992074\\n    ],\\n    [\\n      -76.93381766666667,\\n      38.9921955\\n    ]\\n  ],\\n  "type": "MultiPoint"\\n}\',\n       \'{\\n  "coordinates": [\\n    [\\n      -76.940231,\\n      38.99637666666667\\n    ],\\n    [\\n      -76.940162,\\n      38.996216833333335\\n    ],\\n    [\\n      -76.940121,\\n      38.99595333333333\\n    ],\\n    [\\n      -76.94012633333334,\\n      38.99570416666667\\n    ],\\n    [\\n      -76.9401345,\\n      38.99541283333333\\n    ],\\n    [\\n      -76.94023733333333,\\n      38.99504433333333\\n    ],\\n    [\\n      -76.94017533333333,\\n      38.9948695\\n    ],\\n    [\\n      -76.9399655,\\n      38.99482666666667\\n    ],\\n    [\\n      -76.93987316666667,\\n      38.99473183333333\\n    ],\\n    [\\n      -76.93984583333334,\\n      38.994471\\n    ],\\n    [\\n      -76.93986583333333,\\n      38.99413916666666\\n    ],\\n    [\\n      -76.9398115,\\n      38.993832\\n    ],\\n    [\\n      -76.93961983333334,\\n      38.99374566666667\\n    ],\\n    [\\n      -76.93964633333333,\\n      38.99357916666666\\n    ],\\n    [\\n      -76.93985166666667,\\n      38.993465666666665\\n    ],\\n    [\\n      -76.93990633333334,\\n      38.9933055\\n    ],\\n    [\\n      -76.939865,\\n      38.99308\\n    ],\\n    [\\n      -76.93989633333334,\\n      38.992800333333335\\n    ],\\n    [\\n      -76.940083,\\n      38.99247416666667\\n    ],\\n    [\\n      -76.94025816666667,\\n      38.992195\\n    ],\\n    [\\n      -76.94048083333334,\\n      38.9918425\\n    ],\\n    [\\n      -76.94066466666666,\\n      38.99157116666667\\n    ],\\n    [\\n      -76.940787,\\n      38.9914375\\n    ],\\n    [\\n      -76.941057,\\n      38.99142916666667\\n    ],\\n    [\\n      -76.9413645,\\n      38.9914245\\n    ],\\n    [\\n      -76.9416675,\\n      38.99142483333333\\n    ],\\n    [\\n      -76.94196633333334,\\n      38.99145016666667\\n    ],\\n    [\\n      -76.94230566666667,\\n      38.9914625\\n    ],\\n    [\\n      -76.942615,\\n      38.991445\\n    ],\\n    [\\n      -76.94295816666667,\\n      38.991455333333334\\n    ],\\n    [\\n      -76.94337666666667,\\n      38.9914745\\n    ],\\n    [\\n      -76.943746,\\n      38.991484166666666\\n    ],\\n    [\\n      -76.94408783333333,\\n      38.99158466666667\\n    ],\\n    [\\n      -76.94421416666667,\\n      38.99191166666667\\n    ],\\n    [\\n      -76.944239,\\n      38.99223516666667\\n    ],\\n    [\\n      -76.94421066666666,\\n      38.992535833333335\\n    ],\\n    [\\n      -76.94421416666667,\\n      38.992801166666666\\n    ],\\n    [\\n      -76.94432566666667,\\n      38.99291733333333\\n    ],\\n    [\\n      -76.94447316666667,\\n      38.992935333333335\\n    ],\\n    [\\n      -76.94476683333333,\\n      38.99299933333333\\n    ],\\n    [\\n      -76.94520816666666,\\n      38.99301533333333\\n    ],\\n    [\\n      -76.94546366666667,\\n      38.992949\\n    ],\\n    [\\n      -76.94559966666667,\\n      38.99284216666667\\n    ]\\n  ],\\n  "type": "MultiPoint"\\n}\',\n       \'{\\n  "coordinates": [\\n    [\\n      -76.9441525,\\n      38.991955833333336\\n    ],\\n    [\\n      -76.94412166666666,\\n      38.992103166666666\\n    ],\\n    [\\n      -76.94414666666667,\\n      38.9922205\\n    ],\\n    [\\n      -76.944111,\\n      38.99236616666666\\n    ],\\n    [\\n      -76.94417416666667,\\n      38.9924795\\n    ],\\n    [\\n      -76.94421466666667,\\n      38.992673833333335\\n    ],\\n    [\\n      -76.944087,\\n      38.99279883333333\\n    ],\\n    [\\n      -76.94381483333333,\\n      38.99288116666666\\n    ],\\n    [\\n      -76.94346783333333,\\n      38.99292116666667\\n    ],\\n    [\\n      -76.9431255,\\n      38.99290116666667\\n    ],\\n    [\\n      -76.94284933333333,\\n      38.9929175\\n    ],\\n    [\\n      -76.94253683333334,\\n      38.99291216666667\\n    ],\\n    [\\n      -76.94211766666666,\\n      38.99289616666667\\n    ],\\n    [\\n      -76.94186783333333,\\n      38.99273633333333\\n    ],\\n    [\\n      -76.9418115,\\n      38.99248866666667\\n    ],\\n    [\\n      -76.94182033333334,\\n      38.9922015\\n    ],\\n    [\\n      -76.94187483333333,\\n      38.991979666666666\\n    ],\\n    [\\n      -76.9419145,\\n      38.99184183333333\\n    ],\\n    [\\n      -76.94186766666667,\\n      38.99160583333333\\n    ],\\n    [\\n      -76.9418965,\\n      38.991486\\n    ],\\n    [\\n      -76.941962,\\n      38.99139866666667\\n    ],\\n    [\\n      -76.9416545,\\n      38.991374\\n    ],\\n    [\\n      -76.94129733333334,\\n      38.9913685\\n    ],\\n    [\\n      -76.94100183333333,\\n      38.991386166666665\\n    ],\\n    [\\n      -76.94085233333334,\\n      38.991382\\n    ],\\n    [\\n      -76.94072383333334,\\n      38.991205\\n    ],\\n    [\\n      -76.94071566666666,\\n      38.991043833333336\\n    ],\\n    [\\n      -76.9407295,\\n      38.99084283333333\\n    ],\\n    [\\n      -76.9407055,\\n      38.990614\\n    ],\\n    [\\n      -76.940727,\\n      38.99032966666667\\n    ],\\n    [\\n      -76.94072066666666,\\n      38.990066666666664\\n    ],\\n    [\\n      -76.94070116666667,\\n      38.989818666666665\\n    ],\\n    [\\n      -76.94070266666667,\\n      38.989566333333336\\n    ],\\n    [\\n      -76.940708,\\n      38.98934933333334\\n    ],\\n    [\\n      -76.94069983333333,\\n      38.98915266666667\\n    ],\\n    [\\n      -76.9406875,\\n      38.988945\\n    ],\\n    [\\n      -76.94064516666667,\\n      38.988672\\n    ],\\n    [\\n      -76.940647,\\n      38.988418\\n    ],\\n    [\\n      -76.940625,\\n      38.98815183333333\\n    ],\\n    [\\n      -76.9405185,\\n      38.98788916666667\\n    ],\\n    [\\n      -76.94050583333333,\\n      38.987684\\n    ],\\n    [\\n      -76.9403455,\\n      38.98751683333333\\n    ],\\n    [\\n      -76.94027983333334,\\n      38.98732433333333\\n    ],\\n    [\\n      -76.9403845,\\n      38.987405\\n    ],\\n    [\\n      -76.94025416666666,\\n      38.987308\\n    ],\\n    [\\n      -76.94021583333334,\\n      38.987198666666664\\n    ]\\n  ],\\n  "type": "MultiPoint"\\n}\',\n       \'{\\n  "coordinates": [\\n    [\\n      -76.93416666666667,\\n      38.99064533333333\\n    ],\\n    [\\n      -76.9344025,\\n      38.990251\\n    ],\\n    [\\n      -76.93473616666667,\\n      38.99026216666667\\n    ],\\n    [\\n      -76.93553883333334,\\n      38.990102666666665\\n    ],\\n    [\\n      -76.936415,\\n      38.98961716666667\\n    ],\\n    [\\n      -76.937055,\\n      38.98974916666667\\n    ],\\n    [\\n      -76.93714483333333,\\n      38.9895925\\n    ],\\n    [\\n      -76.93672333333333,\\n      38.98872716666666\\n    ],\\n    [\\n      -76.93579216666667,\\n      38.988570333333335\\n    ],\\n    [\\n      -76.9356455,\\n      38.988189\\n    ],\\n    [\\n      -76.93563016666667,\\n      38.988181166666664\\n    ],\\n    [\\n      -76.9354865,\\n      38.98872516666667\\n    ],\\n    [\\n      -76.93633233333334,\\n      38.98871533333333\\n    ],\\n    [\\n      -76.93687466666667,\\n      38.989182666666665\\n    ],\\n    [\\n      -76.93733383333333,\\n      38.990109833333335\\n    ],\\n    [\\n      -76.93833783333334,\\n      38.9901655\\n    ],\\n    [\\n      -76.93985933333333,\\n      38.990195166666666\\n    ],\\n    [\\n      -76.94067233333334,\\n      38.98968433333334\\n    ],\\n    [\\n      -76.94065983333333,\\n      38.988509\\n    ],\\n    [\\n      -76.94096716666667,\\n      38.9876725\\n    ],\\n    [\\n      -76.94131433333334,\\n      38.98712033333334\\n    ],\\n    [\\n      -76.9412955,\\n      38.9863855\\n    ],\\n    [\\n      -76.941708,\\n      38.98566616666667\\n    ],\\n    [\\n      -76.94236266666667,\\n      38.985197666666664\\n    ],\\n    [\\n      -76.94243216666666,\\n      38.984597\\n    ],\\n    [\\n      -76.94232383333333,\\n      38.9839105\\n    ],\\n    [\\n      -76.9426145,\\n      38.983440333333334\\n    ],\\n    [\\n      -76.9431495,\\n      38.9830885\\n    ],\\n    [\\n      -76.94352133333334,\\n      38.982485\\n    ],\\n    [\\n      -76.94421733333333,\\n      38.982467\\n    ],\\n    [\\n      -76.94495483333333,\\n      38.982380666666664\\n    ],\\n    [\\n      -76.94471683333333,\\n      38.981971333333334\\n    ],\\n    [\\n      -76.9436035,\\n      38.982006166666665\\n    ],\\n    [\\n      -76.9434915,\\n      38.982599\\n    ],\\n    [\\n      -76.94279633333333,\\n      38.982486\\n    ],\\n    [\\n      -76.94393383333333,\\n      38.98235966666667\\n    ],\\n    [\\n      -76.943997,\\n      38.98246133333333\\n    ],\\n    [\\n      -76.94489683333333,\\n      38.982423\\n    ],\\n    [\\n      -76.94623633333333,\\n      38.982509\\n    ],\\n    [\\n      -76.94655866666666,\\n      38.98300416666667\\n    ],\\n    [\\n      -76.94650866666667,\\n      38.98289916666667\\n    ],\\n    [\\n      -76.946411,\\n      38.982194666666665\\n    ],\\n    [\\n      -76.94713116666667,\\n      38.98234583333333\\n    ],\\n    [\\n      -76.94779833333334,\\n      38.982841\\n    ],\\n    [\\n      -76.94828516666666,\\n      38.9836515\\n    ],\\n    [\\n      -76.94861366666666,\\n      38.98440816666667\\n    ],\\n    [\\n      -76.948994,\\n      38.98477583333333\\n    ],\\n    [\\n      -76.948958,\\n      38.98585033333333\\n    ],\\n    [\\n      -76.94894966666666,\\n      38.987113666666666\\n    ],\\n    [\\n      -76.9474575,\\n      38.9871755\\n    ],\\n    [\\n      -76.94585166666667,\\n      38.987247333333336\\n    ],\\n    [\\n      -76.945035,\\n      38.986718833333335\\n    ],\\n    [\\n      -76.94463466666667,\\n      38.985607333333334\\n    ],\\n    [\\n      -76.94361866666667,\\n      38.98546833333333\\n    ],\\n    [\\n      -76.9422605,\\n      38.985444333333334\\n    ],\\n    [\\n      -76.941298,\\n      38.985094833333335\\n    ],\\n    [\\n      -76.94142766666667,\\n      38.984426166666665\\n    ],\\n    [\\n      -76.941689,\\n      38.983709\\n    ],\\n    [\\n      -76.94226933333333,\\n      38.98366033333333\\n    ],\\n    [\\n      -76.94296416666667,\\n      38.983531\\n    ],\\n    [\\n      -76.94220016666667,\\n      38.98346683333333\\n    ],\\n    [\\n      -76.94185216666666,\\n      38.983115833333336\\n    ],\\n    [\\n      -76.941603,\\n      38.98274933333333\\n    ],\\n    [\\n      -76.94153816666666,\\n      38.983334\\n    ],\\n    [\\n      -76.94191266666667,\\n      38.9836315\\n    ],\\n    [\\n      -76.9422025,\\n      38.98371683333333\\n    ],\\n    [\\n      -76.9426895,\\n      38.98352916666666\\n    ],\\n    [\\n      -76.9432255,\\n      38.9835005\\n    ],\\n    [\\n      -76.94365416666666,\\n      38.9834705\\n    ],\\n    [\\n      -76.94425216666667,\\n      38.983600833333334\\n    ],\\n    [\\n      -76.9448005,\\n      38.98406033333333\\n    ],\\n    [\\n      -76.94524633333333,\\n      38.984413\\n    ],\\n    [\\n      -76.94575283333333,\\n      38.984788\\n    ],\\n    [\\n      -76.94622516666666,\\n      38.98507566666667\\n    ],\\n    [\\n      -76.94662183333334,\\n      38.98511583333333\\n    ],\\n    [\\n      -76.94751616666667,\\n      38.98493416666667\\n    ],\\n    [\\n      -76.94838333333334,\\n      38.984913666666664\\n    ],\\n    [\\n      -76.94895333333334,\\n      38.984828\\n    ],\\n    [\\n      -76.94884533333334,\\n      38.98458433333333\\n    ],\\n    [\\n      -76.94866633333334,\\n      38.98485066666667\\n    ],\\n    [\\n      -76.9489825,\\n      38.98508966666667\\n    ],\\n    [\\n      -76.94903616666667,\\n      38.98568066666667\\n    ],\\n    [\\n      -76.949013,\\n      38.986324\\n    ],\\n    [\\n      -76.948972,\\n      38.9869615\\n    ],\\n    [\\n      -76.94902033333334,\\n      38.98759883333334\\n    ],\\n    [\\n      -76.94915233333333,\\n      38.9882305\\n    ],\\n    [\\n      -76.94898083333334,\\n      38.988803\\n    ],\\n    [\\n      -76.94920333333333,\\n      38.989193\\n    ],\\n    [\\n      -76.94939033333333,\\n      38.9897825\\n    ],\\n    [\\n      -76.9493975,\\n      38.9904165\\n    ],\\n    [\\n      -76.94938666666667,\\n      38.9910555\\n    ],\\n    [\\n      -76.94927933333334,\\n      38.99168383333333\\n    ],\\n    [\\n      -76.94881183333334,\\n      38.99216933333334\\n    ],\\n    [\\n      -76.94808216666667,\\n      38.992182166666666\\n    ],\\n    [\\n      -76.947537,\\n      38.991745\\n    ],\\n    [\\n      -76.94693733333334,\\n      38.99131583333333\\n    ],\\n    [\\n      -76.94624916666666,\\n      38.99145683333333\\n    ],\\n    [\\n      -76.9459925,\\n      38.99168916666667\\n    ],\\n    [\\n      -76.9456165,\\n      38.992202166666665\\n    ],\\n    [\\n      -76.945576,\\n      38.9927255\\n    ],\\n    [\\n      -76.94561683333333,\\n      38.992582\\n    ],\\n    [\\n      -76.9456955,\\n      38.992067\\n    ],\\n    [\\n      -76.94616183333333,\\n      38.9915805\\n    ],\\n    [\\n      -76.94644566666666,\\n      38.9910955\\n    ],\\n    [\\n      -76.9457535,\\n      38.99072366666667\\n    ],\\n    [\\n      -76.94492983333333,\\n      38.990574333333335\\n    ],\\n    [\\n      -76.944108,\\n      38.990583\\n    ],\\n    [\\n      -76.94330333333333,\\n      38.99058383333333\\n    ],\\n    [\\n      -76.94272333333333,\\n      38.9905865\\n    ],\\n    [\\n      -76.94208166666667,\\n      38.990586\\n    ],\\n    [\\n      -76.94134383333333,\\n      38.99031166666666\\n    ],\\n    [\\n      -76.94075816666667,\\n      38.990027833333336\\n    ],\\n    [\\n      -76.94069083333333,\\n      38.989415666666666\\n    ],\\n    [\\n      -76.94065983333333,\\n      38.98878466666667\\n    ],\\n    [\\n      -76.94064016666667,\\n      38.98815583333333\\n    ],\\n    [\\n      -76.94044483333333,\\n      38.98757233333333\\n    ],\\n    [\\n      -76.9401695,\\n      38.9869875\\n    ],\\n    [\\n      -76.9401735,\\n      38.98639\\n    ],\\n    [\\n      -76.9401535,\\n      38.985743\\n    ],\\n    [\\n      -76.940141,\\n      38.985120333333334\\n    ],\\n    [\\n      -76.94013466666667,\\n      38.984638\\n    ],\\n    [\\n      -76.94009083333333,\\n      38.984104\\n    ],\\n    [\\n      -76.94001,\\n      38.983456\\n    ],\\n    [\\n      -76.939795,\\n      38.982875166666666\\n    ],\\n    [\\n      -76.93914883333333,\\n      38.982461666666666\\n    ],\\n    [\\n      -76.938329,\\n      38.982265\\n    ],\\n    [\\n      -76.93806066666667,\\n      38.98218283333333\\n    ],\\n    [\\n      -76.93773566666667,\\n      38.982138666666664\\n    ],\\n    [\\n      -76.9373385,\\n      38.98207383333333\\n    ],\\n    [\\n      -76.937194,\\n      38.982088166666664\\n    ],\\n    [\\n      -76.9368345,\\n      38.98208416666667\\n    ],\\n    [\\n      -76.93658383333333,\\n      38.98206233333333\\n    ],\\n    [\\n      -76.9364265,\\n      38.982063333333336\\n    ],\\n    [\\n      -76.936361,\\n      38.98225766666667\\n    ],\\n    [\\n      -76.93629683333333,\\n      38.98240233333333\\n    ],\\n    [\\n      -76.93617,\\n      38.98238416666667\\n    ],\\n    [\\n      -76.93599783333333,\\n      38.9823935\\n    ],\\n    [\\n      -76.93593566666667,\\n      38.98253233333333\\n    ],\\n    [\\n      -76.93608833333333,\\n      38.9824515\\n    ]\\n  ],\\n  "type": "MultiPoint"\\n}\'],\n      dtype=\'object\', length=40078)] are in the [columns]'